### PR TITLE
feat: integrate wallet connect

### LINKED ISSUE #52: Evaluate using WalletConnect along with Metamask for crypto payments
**Is your feature request related to a problem? Please describe.**
Evaluate how much effort is required to support WalletConnect along with MetaMask for crypto payment.

### PR DESCRIPTION
**Pull request by lukicenturi**
Closes #52

### CODE REVIEW

File: store/index.ts [hunk 1]
```diff
@@ -396,7 +396,7 @@ export const useMainStore = defineStore('main', () => {
   ): Promise<Result<CryptoPayment, PaymentError>> => {
     try {
       const response = await fetchWithCsrf<CryptoPaymentResponse>(
-        '/webapi/payment/crypto',
+        '/webapi/payment/crypto/',
```
Comment (kelsos) [inline store/index.ts:399]: Hmm does this call work on production?

something tells me this is one of the changes related to #241

File: locales/en.json [hunk 2]
```diff
@@ -551,7 +551,9 @@
           },
           "metamask": {
             "action": "Pay with Metamask",
```
Comment (kelsos) [inline locales/en.json:553]: Do we use this does it still make sense to have it as `pay with metamask`?

File: store/index.ts (hunk 1 above)
Comment (lukicenturi) [reply store/index.ts]: reverted

File: locales/en.json (hunk 2 above)
Comment (lukicenturi) [reply locales/en.json:554]: updated now

**Pull request merged**